### DISCUSSION TITLE
Need help in understanding load cases

### DISCUSSION THREAD

**Discussion by lokitkhemka** [xcfem/xc#134] [Q&A] [answered]
I am going through the `slope_stability_analysis_example_01` and  I have questions about how the loads are defined

```
ts= modelSpace.newTimeSeries(name= 'ts', tsType= "path_time_ts")
ts.path= xc.Vector([0, 1])
ts.time= xc.Vector([0, 100])

lp0= modelSpace.newLoadPattern(name= '0')
modelSpace.setCurrentLoadPattern(lp0.name)
accel= xc.Vector([0,g])
for e in s1.elements:
    e.createInertiaLoad(accel)
modelSpace.addLoadCaseToDomain(lp0.name)
```
`ts` is applying a gradually increasing load starting from `0` to `1` (N or kN?) in `+Y` or '-Y` direction from time 0 to 100? Also, which nodes is `ts` applied to? All the nodes?

Thank you for your help.

**Comment by lcpt** [comment, marked as answer]
Hi, @lokitkhemka.

The time series object `ts` determines the factor that multiplies the loads in the load pattern (`lp0`). From the definition of its path, it returns a factor equal to zero when the pseudo-time is equal to zero and a factor equal to one when the pseudo-time is equal to 100. You can take a look at the examples [in this directory](https://github.com/xcfem/xc/tree/master/verif/tests/loads/time_series) to see how this works.

The inertia load is applied to the elements of `s1` when the for loop is executed:
```
for e in s1.elements:
    e.createInertiaLoad(accel)
```

Just for reference (maybe other users want to know) the `slope_stability_analysis_example_01.py` can be found [here](https://github.com/xcfem/xc_examples/blob/master/geotechnics/slope_stability_analysis/slope_stability_analysis_example_01.py).

**Reply by lokitkhemka** [reply]
Understood. Basically, the inertial load is increased from `0.0 * gamma` to `1.0 * gamma` in the time period of `100`. Thank you so much.

**Reply by lcpt** [reply]
With pleasure.
